versions.tf

```diff
@@ -1,26 +1,26 @@
 terraform {
-  required_version = ">= 1.3.7"
+  required_version = ">= 1.5.4"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.0"
+      version = "5.68.0"
```

Comment:
i pinned at this version because later versions were proving to be problematic during local testing.